wpilibj/src/main/java/edu/wpi/first/wpilibj/apriltag/AprilTag.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.apriltag;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.wpi.first.math.geometry.Pose3d;
+import java.util.Objects;
+
+public class AprilTag {
+  @SuppressWarnings("MemberName")
+  @JsonProperty(value = "ID")
+  public int m_ID;
+
+  @JsonProperty(value = "pose")
+  public Pose3d m_poseMeters;
```

Comment:
We don't append units for public member variables. In fact, these should be named "ID" and "pose" to match C++. We only include the `m_` on private member variables. You maybe have to suppress a MemberName warning when doing that, but you can add that to the class itself instead of repeating it for each member variable.